system-variables.md

```diff
@@ -1680,19 +1793,22 @@ SET tidb_slow_log_threshold = 200;
 ### tidb_store_limit <span class="version-mark">New in v3.0.4 and v4.0</span>
 
 - Scope: GLOBAL
+- Persists to cluster: Yes
 - Default value: `0`
 - Range: `[0, 9223372036854775807]`
 - This variable is used to limit the maximum number of requests TiDB can send to TiKV at the same time. 0 means no limit.
 
-### tidb_sysdate_is_now （New in v6.0.0）
```

Comment:
`(New in v6.0.0)` is a makeshift format. If a heading with `<span class="version-mark">` is referenced in an anchor, even if it passes the CI link check, the <docs.pingcap.com> site no longer parses the anchor as it always did, probably by a frontend misoperation. Maybe they will fix it in the future. So `(New in v6.0.0)` is a makeshift format that makes sure the anchor works and also makes the CI happy.

Reply:
If we write the heading as `### tidb_sysdate_is_now <span class="version-mark">New in v6.0.0</span>` as we did before, and reference it in an anchor `[](/system-variables.md#tidb_sysdate_is_now-new-in-v600)`, when the docs site builds is ready and you click on the referenced anchor link, you **will not** be directed to the exact heading position.

Reply:
It is inconsistent with the previous format though (and looks weird).

I can change them all to be in the format:

```
### VariableName
- Introduced: {Version Number}
- Default value: `0`
- Range: `[x, y]`
```

This would make the anchors more stable, but I would suggest we do it in a followup PR because they'll all change.

Reply:
Good idea!